tests/unit/test_api_urls.py

```diff
@@ -94,7 +94,8 @@ def test_api_labels_without_auth(self):
                 {'status_code': 200, 'content': '[]'})
             rv = self.app.get('/api/issues/labels', environ_base=headers)
             self.assertEqual(rv.status_code, 200)
-            self.assertEqual(rv.content_type, 'application/json')
+            self.assertEqual(
+                rv.content_type, 'application/vnd.github.v3.html+json')
```

Comment:
as we are testing the label json only

Reply:
> So I hope I'm wrong, but I guess many fixtures are not worth fixing. 

@karlcow If we don't leave as `body_html` we won't be able to visualize the issue. If we go to webcompat.com/issues/100 it will show a blank page... So that's why I changed all of the files.
Why do you think we should keep them in the old way?

Reply:
this is a request to `'/api/issues/labels'` so not related to body_html. Probably this comment is a reply to something else. :)
The request to /api/issues/labels should definitely stays as `'application/json'` see the other discussion about mime type. :)